dist/t/spec/spec_helper.rb

```diff
@@ -19,6 +18,7 @@
   end
   # Limits the available syntax to the non-monkey patched
   config.disable_monkey_patching!
+  config.include Capybara::DSL
```

Comment:
It should apply magically only on features: http://stackoverflow.com/questions/18616683/why-do-i-get-including-capybaradsl-in-the-global-scope-is-not-recommended?answertab=votes#tab-top